app/uk/gov/hmrc/controllers/HeaderValidator.scala

```diff
@@ -43,7 +40,7 @@ trait HeaderValidator extends Results {
   def validateAccept(rules: Option[String] => Boolean): ActionBuilder[Request, AnyContent] = new ActionBuilder[Request, AnyContent] {
     def invokeBlock[A](request: Request[A], block: Request[A] => Future[Result]): Future[Result] = {
       if (rules(request.headers.get(ACCEPT))) block(request)
-      else Future.successful(Status(ErrorAcceptHeaderInvalid.httpStatusCode)(Json.toJson(ErrorAcceptHeaderInvalid)))
+      else Future.successful(Status(ErrorAcceptHeaderInvalid.httpStatusCode))
```

Comment:
Is this deliberate to drop the body of the response?

Reply:
good catch, didn't want to remove